README.md

```diff
@@ -153,10 +160,27 @@ await assetCompute.register();
 sleep(45000); // sleep after registering to give time for journal to set up
 await assetCompute.process(..renditions);
 ```
+### Using custom 429 retry options
+By default, `AssetComputeClient` will retry 4 times (with smart backpressure) on 429s.
+
+Retry 10 times:
+```js
+const assetCompute = new AssetComputeClient(integration, {
+    max429RetryCount: 10
+});
+```
+Disable retry:
+```js
+const assetCompute = new AssetComputeClient(integration, {
+    disable429Retry: 10
```

Comment:
Should this be `true`?

Reply:
yep, nice catch. I also made the optimization changes you recommended for the error class